extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzRuntimeConfig.java

```diff
@@ -33,4 +33,12 @@
     @ConfigItem
     public boolean forceStart;
 
+    /**
+     * Scheduler will immediately start running if it finds a {@link io.quarkus.scheduler.Scheduled} business method
+     * or is forced to start using forceStart property.
+     * If set to true the scheduler will be not start triggering jobs until an explicit start is called from the main
+     * scheduler. This is useful to programmatically register listeners before scheduler starts performing some work.
+     */
+    @ConfigItem
+    public boolean haltStart;
```

Comment:
I wonder if we should introduce a new config property, e.g. `startMode` with values `normal`, `forced` and `halted` and deprecate the `forceStart` instead. WDYT?

CC @machi1990

Reply:
+1 Great thinking.

Reply:
LGTM